packages/react-examples/src/react-charting/VerticalBarChart/VerticalBarChart.AxisTitle.Example.tsx

```diff
@@ -0,0 +1,220 @@
+import * as React from 'react';
```

Comment:
This example isn't showing in the demo site

Reply:
now showing